rdwatch/core/views/animation.py

```diff
@@ -0,0 +1,218 @@
+                    'taskId': export.celery_id,
+                    'state': celery_data,
+                }
+            )
+    return output
+
+
+@router.post('/site/{id}/')
+def generate_site_animation(
+    request: HttpRequest,
+    id: UUID4,
+    params: GenerateAnimationSchema,  # noqa: B008
+):
+    # Fetch the SiteEvaluation instance
+    task_id = create_site_animation_export.delay(
+        site_evaluation_id=id, settings=params.dict(), userId=request.user.pk
+    )
+    return task_id.id
+
+
+@router.get('/site/{id}/downloads/')
+def get_site_downloads(
+    request: HttpRequest,
+    id: UUID4,
+):
+    exports = AnimationSiteExport.objects.filter(
+        site_evaluation=id, user=request.user
+    ).order_by('-created')
+    return generate_download_data(exports)
+
+
+@router.get('/modelrun/{id}/downloads/')
+def get_modelrun_downloads(
+    request: HttpRequest,
+    id: UUID4,
+):
+    exports = AnimationModelRunExport.objects.filter(
+        configuration=id, user=request.user
+    ).order_by('-created')
+    return generate_download_data(exports)
+
+
+class DeleteErrorSchema(BaseModel):
+    error: str
+
+
+class DeleteSuccessSchema(BaseModel):
+    success: str
+
+
+@router.delete(
+    '/site/{id}/',
+    response={
+        403: DeleteErrorSchema,
+        200: DeleteSuccessSchema,
+    },
+)
+def delete_site_download(
+    request: HttpRequest,
+    id: UUID4,
+):
+    # Fetch the export object
+    export = get_object_or_404(AnimationSiteExport, celery_id=id, user=request.user)
+    logger.warning(export)
+    # Check for ownership or permissions
+    if export.user != request.user:
+        return 403, {'error': 'You do not have permission to delete this export.'}
+
+    # Perform the delete operation
+    export.delete()
+    return 200, {'success': 'Site Animation Export deleted successfully.'}
+
+
+@router.delete(
+    '/modelrun/{id}/',
+    response={
+        403: DeleteErrorSchema,
+        200: DeleteSuccessSchema,
+    },
+)
+def delete_modelrun_download(
+    request: HttpRequest,
+    id: UUID4,
+):
+    # Fetch the export object
+    export = get_object_or_404(AnimationModelRunExport, celery_id=id, user=request.user)
+
+    # Perform the delete operation
+    export.delete()
+    return 200, {'success': 'ModelRun Animation Export deleted successfully.'}
+
+
+@router.post('/site/{id}/debug/')
+def generate_animation_debug(
+    request: HttpRequest,
+    id: UUID4,
+    params: GenerateAnimationSchema,  # noqa: B008
+):
+    # Fetch the SiteEvaluation instance
+    datapath, name = create_animation(
+        site_evaluation_id=id,
+        settings=params.dict(),
+    )
+    if datapath:
+        with open(datapath, 'rb') as f:
+            response = HttpResponse(f.read(), content_type='application/octet-stream')
+            response['Content-Disposition'] = f'attachment; filename={name}'
+            return response
+    return 500, 'Unable to export data'
```

Comment:
```suggestion
```

Reply:
I'll remove this before we merge.  Once we have images again this is a quick endpoint to utilize if you need to test any changes to the animation drawing and don't want to deal with celery container reloading.